### PR TITLE
Fuel ECAM visual adjustments and center/APU pump indicator fix

### LINKED ISSUE #28: ECAM FUEL page is wrong
SD FUEL 320 NEO

The A320 NEO has the jetpumps of the A321.

### PR DESCRIPTION
**Pull request by wpine215**
Fixes #28 

**Summary of Changes**

- Adjusted size, layout, and position of multiple elements to match real fuel ECAM page on A320neo
- Fixed issue with APU indicator not accurately reflecting APU pump status
- Fixed wrong center pump indicators (mentioned in issue above)

**Screenshots (if necessary)**

Before:
FlightSimulator_d6kVKGiXuf

After:
FlightSimulator_KYKqyXqHKg

**References**

Referenced documents provided by Ares.

image

**Additional context**

Discord username (if different from GitHub): Iceman#5540

### CODE REVIEW

**Comment by camilo625** [comment]
Name: Camilo625 #5575
Date of testing: Current date 04/10/2020
Version of the sim: 1.9.3.0
PR Tested: #1074 

Testing technique: Spawn and test pumps
Steps you've made to test the Pull Request:
1. Spawn C&B
2. Test pumps
3. Test crossfeed

Overall Rating: Good
Conclusions of the testing: Works as described, no issues found

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_Fuel.css [hunk 1]
```diff
@@ -41,47 +41,54 @@
...
-    top: -25%;
-    left: -12.5%;
-    width: 125%;
-    height: 125%; }
+    position: absolute;
+    top: -5%;
+    bottom: 5%;
+    left: 0%;
+    width: 100%;
+    height: 100%; }
```
Comment (Meqolo) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_Fuel.css:49]: Place the closing bracket on the next line, its more readable

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_Fuel.css [hunk 2]
```diff
@@ -41,47 +41,54 @@
...
     a320-neo-lower-ecam-fuel .MainShape {
       fill: transparent;
       stroke: lightgray;
-      stroke-width: 2; }
+      stroke-width: 5; }
```
Comment (Meqolo) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_Fuel.css:53]: Place the closing bracket on the next line, its more readable

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_Fuel.css [hunk 3]
```diff
@@ -41,47 +41,54 @@
...
     a320-neo-lower-ecam-fuel .FlowShape {
       fill: rgba(13,20,35,1);
       stroke: green;
-      stroke-width: 2; }
+      stroke-width: 3; }
```
Comment (Meqolo) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_Fuel.css:57]: Place the closing bracket on the next line, its more readable

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_Fuel.css [hunk 4]
```diff
@@ -41,47 +41,54 @@
...
     a320-neo-lower-ecam-fuel .ValvePumpActive {
       fill: rgba(13,20,35,1);
       stroke: green;
-      stroke-width: 2; }
+      stroke-width: 3; }
```
Comment (Meqolo) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_Fuel.css:61]: Place the closing bracket on the next line, its more readable

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_Fuel.css [hunk 5]
```diff
@@ -41,47 +41,54 @@
...
     a320-neo-lower-ecam-fuel .ValvePumpInactive {
       fill: rgba(13,20,35,1);
       stroke: #db7200;
-      stroke-width: 2; }
+      stroke-width: 3; }
```
Comment (Meqolo) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_Fuel.css:65]: Place the closing bracket on the next line, its more readable

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_Fuel.css [hunk 6]
```diff
@@ -41,47 +41,54 @@
...
     a320-neo-lower-ecam-fuel text {
-      font-family: Roboto-Bold; }
+      font-family: Roboto-Bold;
+      font-size: 20px; }
```
Comment (Meqolo) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_Fuel.css:68]: Place the closing bracket on the next line, its more readable

**Pull request merged**